--- Makefile.orig	2015-06-14 20:12:31.000000000 +0100
+++ Makefile	2015-06-14 20:12:46.000000000 +0100
@@ -1,4 +1,4 @@
-include ../make.inc  #allow auto configuration of CC
+#include ../make.inc  #allow auto configuration of CC
 DEST_DIR = 	../exec
 DEST=		${DEST_DIR}/chaco
 #CC = 		gcc
@@ -97,6 +97,9 @@
 lint:
 		lint ${IFLAG} ${FILES.c} -lm
 
+libchaco:	${DEST}
+		$(AR) $(ARFLAGS) ${DEST_DIR}/libchaco.a ${FILES.o}
+
 alint:
 		alint ${IFLAG} ${FILES.c} -lm
 
