--- config/BuildSystem/config/packages/ctetgen.py.orig	2015-06-08 23:14:54.000000000 +0100
+++ config/BuildSystem/config/packages/ctetgen.py	2015-06-15 01:40:29.000000000 +0100
@@ -38,7 +38,7 @@
       self.log.write(output+err)
       self.logPrintBox('Installing Ctetgen; this may take several minutes')
       self.installDirProvider.printSudoPasswordMessage(1)
-      output,err,ret  = config.package.GNUPackage.executeShellCommand('cd '+self.packageDir+' && '+self.installDirProvider.installSudo+self.make.make+' PETSC_DIR='+self.petscdir.dir+' install-ctetgen',timeout=1000, log = self.log)
+      output,err,ret  = config.package.GNUPackage.executeShellCommand('cd '+self.packageDir+' && '+self.make.make+' PETSC_DIR='+self.petscdir.dir+' DESTDIR='+self.petscdir.dir+' install-ctetgen',timeout=1000, log = self.log)
       self.log.write(output+err)
     except RuntimeError, e:
       raise RuntimeError('Error running make on Ctetgen: '+str(e))
